ant-cli/src/actions/connect.rs

```diff
@@ -7,13 +7,17 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 use crate::network::NetworkPeers;
+use autonomi::client::config::ClientOperationConfig;
 use autonomi::{get_evm_network, Client, ClientConfig};
 use color_eyre::eyre::bail;
 use color_eyre::eyre::Result;
 use indicatif::ProgressBar;
 use std::time::Duration;
 
-pub async fn connect_to_network(peers: NetworkPeers) -> Result<Client> {
+pub async fn connect_to_network(
+    peers: NetworkPeers,
+    client_operation_config: ClientOperationConfig,
```

Comment:
```suggestion
    client_operation_config: Option<ClientOperationConfig>,
```
Make this an option so we don't need to do the check in every single CLI function that calls this. We can then 
```
client_operation_config.unwrap_or_default()
```